lib/Gemfile

```diff
@@ -32,8 +32,8 @@ gem "memoist",              "~>0.11.0",       :require => false
 gem "more_core_extensions", "~>1.2.0",        :require => false
 gem "nokogiri",             "~>1.5.0",        :require => false
 gem "ovirt",                "~>0.4.2",        :require => false
-gem "kubeclient",           ">=0.1.16",       :require => false
-gem "openshift_client",     ">=0.0.5",        :require => false
+gem "kubeclient",           ">=0.1.17",       :require => false
+gem "openshift_client",     ">=0.0.6",        :require => false
```

Comment:
This may collide with #3090 let's see what gets in first. (Nothing to do yet, let's just be aware of that).